varnish-sys/src/vcl/backend.rs

```diff
@@ -98,8 +96,8 @@ pub struct Backend<S: Serve<T>, T: Transfer> {
     #[expect(dead_code)]
     methods: Box<ffi::vdi_methods>,
     inner: Box<S>,
-    #[expect(dead_code)]
-    type_: CString,
+    #[allow(dead_code)]
```

Comment:
why allow?  expect is better because it will not compile if not needed

Reply:
I think I wrote that one before you moved to `expect`, easy to change